server/app/schema.py

```diff
@@ -315,6 +335,13 @@ def resolve_items(self, info):
         query = Item.get_query(info)
         return query.all()
 
+    # not sure if we'll end up using this but could be useful
+    sets = graphene.NonNull(graphene.List(graphene.NonNull(Set)))

```

Comment:
We should be careful about loading and sending this much data, it can't be good for our server/costs. For development it's OK, but we definitely should turn it off or paginate it when we go to production.

Reply:
Yea I imagine we would be paginating like we would for items. I included it to sort of open up the discussion as to whether or not browsing sets is something desirable that we want to include as a feature.

Reply:
Yeah, I definitely think so, equipping an entire set at once is a very common action and one that's too hidden on Dofusbook I think